File: C/BOJ_5063_TGN.c
#include <stdio.h>
int main()
{
   int n, r, e, c;
   scanf("%d", &n);
   for (int i = 0; i < n; i++)
   {
      scanf("%d %d %d", &r, &e, &c);
      e -= c;        //광고 수익
      if (r < e) printf("advertise\n");
      else if (r > e) printf("do not advertise\n");
      else if (r == e) printf("does not matter\n");
   }
   return 0;
}

File: Python/BOJ_5063_TGN.py
import sys

num = int(sys.stdin.readline())
for i in range(num):
    r, e, c = map(int, sys.stdin.readline().split(' '))
    if (e - c) > r:
        print('advertise')
    elif (e - c) < r:
        print('do not advertise')
    else:
        print('does not matter')